### DISCUSSION TITLE
Expand all json-views of Dataset items etc.

### DISCUSSION THREAD

**Discussion by tkreuder** [langfuse/langfuse#2454] [Ideas]
### Describe the feature or potential improvement

Currently the output is limited to 500 chars, if the medium button is pressed:
![image](https://github.com/langfuse/langfuse/assets/385110/94742c1d-fd14-42eb-9986-ffcd1e6e23aa)

https://github.com/langfuse/langfuse/blob/bdfc51ca72cb31d1a5ae9b2bfacd160e2567cc40/web/src/components/ui/CodeJsonViewer.tsx#L49

Can you add an option or adjust the length to more characters? else each element has to be clicked.

**Comment by marcklingen** [comment]
great idea, thanks for sharing, implemented here: https://github.com/langfuse/langfuse/pull/2626

**Reply by tkreuder** [reply]
Thanks a lot for implementing it 👍  @marcklingen
